core-modular/src/core-api/ux.ts

```diff
@@ -51,6 +54,15 @@ export class UX {
     });
   };
 
+  setDocumentTitle = (documentTitle: string) => {
+    this.documentTitle.set(documentTitle);
+    this.luigi.getEngine()._connector?.setDocumentTitle(documentTitle);
+  };
+
+  getDocumentTitle = (): string => {
+    return get(this.documentTitle) || '';
```

Comment:
I'm getting `''` after i set it. Could you double check if this happens for you as well?